src/__tests__/pages/api/auth/signup.test.ts

```diff
@@ -0,0 +1,238 @@
+/**
+ * @jest-environment node
+ */
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import { PrismaClient } from '@prisma/client';
+import signUpHandler from '../../../../pages/api/auth/signup';
+import {
+  mockUserData,
+  mockManyUserData
+} from '../../../../__mocks__/pages/api/auth';
+import type { UserDataProp } from '../../../../__mocks__/pages/api/auth';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const prisma = new PrismaClient();
+
+beforeAll(async () => {
+  await prisma.$connect();
+});
+
+beforeEach(async () => {
```

Comment:
Does this run before each `describe()` or before each `it()`?

Reply:
before all it